flax/linen/module.py

```diff
@@ -1488,12 +1488,61 @@ def init(self,
            **kwargs) -> FrozenVariableDict:
     """Initializes a module method with variables and returns modified variables.
 
+    ``init`` takes a ``PRNGKey`` as its first argument and all the ``*args`` and ``**kwargs``
+    that the module's ``__call__`` method takes, and returns a dictionary of initialized
```

Comment:
Doesn't have to be `__call__`, it `__call__` by default but if argument `method` is specified it will use that method.

Reply:
fixed